source/upgrade/important-upgrade-notes.rst

```diff
@@ -25,6 +25,12 @@ Important Upgrade Notes
 |                                                    | `Elasticsearch <https://docs.mattermost.com/scale/elasticsearch.html>`_ for autocomplete will have to reindex their data to get the new results. Since this can  |
 |                                                    | take a long time, we suggest disabling autocomplete and running indexing in the background. When this is complete, re-enable autocomplete. Note that only channel|
 |                                                    | members will see private channel names in autocomplete results.                                                                                                  |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Also I would suggest moving this to the top with the bullet points, since this affects more than just the 6.2 version.